Address external debug issue #11 (#20)
riscv/riscv-smmtt: chapter2.adoc

describes CSRs to allow M-mode software to retain control on notification of
interrupts when Supervisor domains are enabled.

* `Smsdedbg` (<<Smsdedbg>>) - This extension defines the external debug
configuration interface that is managed by M-mode software to opt-in to

> external debug when authorized by a RoT.

Reviewer: The M-mode cannot determine whether debug is authorized or not. It should however not need to because Smsdedbg only indicates "Debug Allowed".

Author: Updating this to state - 
* `Smsdedbg` (<<Smsdedbg>>) - This extension defines the external debug
configuration interface that is managed by M-mode software to declare that
external debug is allowed for a supervisor domain. Whether external debug is
authorized or not is expected to be done via a RoT and is outside the scope
of this specification.

describes CSRs to allow M-mode software to retain control on notification of
interrupts when Supervisor domains are enabled.

> * `Smsdedbg` (<<Smsdedbg>>) - This extension defines the external debug
> configuration interface that is managed by M-mode software to declare that
> external debug is allowed for a supervisor domain. Whether external debug is

Author: ```suggestion
* `Smsdedbg` (<<Smsdedbg>>) - This extension provides the controls to indicate
if external debug is allowed for a supervisor domain. Whether external debug is
```

authorized or not is expected to be done via a RoT and is outside the scope
of this specification.

> 

Reviewer: Since the extension is Sm it is already stated that its M-mode.

`Smsdedbg` (<<Smsdedbg>>) - This extension provides the controls to
indicate if external debug is allowed for a supervisor domain.
